docs/src/ScrollableContainer.doc.js

```diff
@@ -624,25 +624,20 @@ function ScrollableContainerExample() {
 );
 
 card(
-  <MainSection
-    name="Related"
-    description={`
-
-[Box](/Box)
-
-Box's overflow prop specifies what should happen if the content is larger than the bounding box. Box should not be replaced with ScrollableContainer if the goal is simply to allow the Box to scroll when content overflows. ScrollableContainer is only needed when anchored components, such as [Tooltip](/Tooltip), [Flyout](/Flyout), [Typeahead](/Typeahead) and [Dropdown](/Dropdown), are used within a container that could potentially scroll.
-
-
-[Modal](/Modal) / [Sheet](/Sheet)
-
-Modal and Sheet come with ScrollableContainer built-in, so any anchored components used in their children tree should work out-of-the-box. Passing an additional ScrollableContainer will break the existing styling on scroll.
-
+  <MainSection name="Related">
+    <MainSection.Subsection
+      description={`
+      **[Box](/Box)**
+      Box's [overflow prop](/Box#Overflow) specifies what should happen if the content is larger than the bounding box. Box should not be replaced with ScrollableContainer if the goal is simply to allow the Box to scroll when content overflows. ScrollableContainer is only needed when anchored components, such as [Tooltip](/Tooltip), [Flyout](/Flyout), [Typeahead](/Typeahead) and [Dropdown](/Dropdown), are used within a container that could potentially scroll.
```

Comment:
Can we monospace "overflow" in "overflow prop", or does that not work with the link?

no article in "to allow the Box to scroll"

s/and/or in the list of anchored components